BXFinance_iOS_source/Sample Code/BXFinance/AppDelegate.swift

```diff
@@ -93,30 +109,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         }
     }
     
-    // added from Verify
```

Comment:
This might be required to use push notifications for MFA. Did you get a chance to test it?